File: Java/1008.java
import java.util.Scanner;

public class Main {

  public static void main(String[] args) {
    Scanner scanner = new Scanner(System.in);

    int A = scanner.nextInt();
    int B = scanner.nextInt();
    double C = scanner.nextDouble();

    double salario = B * C;
    
    String salarioFormatado = String.format("%.2f", salario);

    System.out.println("NUMBER = " + A);
    System.out.println("SALARY = U$ " + salarioFormatado);
  }
}

File: Javascript/1008.js
var input = require('fs').readFileSync('stdin', 'utf8');
var lines = input.split('\n');

var numero = parseInt(lines.shift());
var horasTrabalhadas = parseInt(lines.shift());
var salarioPorHora = parseFloat(lines.shift());

var salario = horasTrabalhadas * salarioPorHora

console.log("NUMBER = " + numero)
console.log("SALARY = U$ " + salario.toFixed(2))